test/Driver/filelists.swift

```diff
@@ -1,3 +1,4 @@
+// UNSUPPORTED: win32
```

Comment:
Why is this one unsupported on Windows?

Reply:
This one uses driver-use-frontend-path, see also https://github.com/apple/swift/pull/21052